constants/roles.js

```diff
@@ -1,10 +1,4 @@
-// Use Legacy Roles with authorizeUsers middleware
-const LEGACY_ROLES = {
-  SUPER_USER: "superUser",
-  APP_OWNER: "appOwner",
-};
-
 // Use Roles with authorizeRoles middleware
 const ROLES = { SUPERUSER: "super_user", APPOWNER: "app_owner", MEMBER: "member", ARCHIVED: "archived" };
```

Comment:
Can we have each key value pair on a new line.

Reply:
made the changes